Updates to the Cloudflare Zaraz docs (#3798)
cloudflare/cloudflare-docs: content/zaraz/web-api/_index.md

# Web API

> Zaraz provides a client-side web API that you can use in your website. This API allows you to send events and data to Zaraz, that you can later use when creating your triggers. Using the API lets you tailor the behavior of Zaraz to your needs: You can launch tools only when you need them, or send information you care about it that isn't otherwise automatically collected from your site.

Reviewer: ```suggestion
Zaraz provides a client-side web API that you can use in your website. This API allows you to send events and data to Zaraz, that you can later use when creating your triggers. Using the API lets you tailor the behavior of Zaraz to your needs: You can launch tools only when you need them, or send information you care about that is not otherwise automatically collected from your site.
```
